NetCasbin/Enforcer.cs

```diff
@@ -631,12 +535,16 @@ private bool Enforce(IReadOnlyList<object> requestValues, ICollection<IEnumerabl
                 IReadOnlyList<string> policyValues = Enumerable.Repeat(string.Empty, policyTokenCount).ToArray();
                 ExpressionHandler.SetPolicyParameters(policyValues);
                 var nowEffect = GetEffect(ExpressionHandler.Invoke(expressionString, requestValues));
-                finalResult = Effector.MergeEffects(effect, new[] { nowEffect }, null, out hitPolicyIndex);
```

Comment:
Actually, I do not quite understand this pr, the old behavior is put all policy evaluate result into an effects array and then decide the final result in `MergeEffects`, you skiped this `MergeEffects` how can you have a whole view of all the policy effects.